### PR TITLE
Support for platforms based on Clounix Networks' device

### PR DESCRIPTION
**Pull request by songqh2022**
**What I did**
Support for platforms based on Clounix Networks' device
**Why I did it**
Support for platforms based on Clounix Networks' device
**How I verified it**
Have verified on the hardware device
**Details if related**

### CODE REVIEW

**Comment by songqh2022** [comment]
retest

File: orchagent/aclorch.cpp [hunk 1]
```diff
@@ -2243,6 +2243,7 @@ void AclOrch::init(vector<TableConnector>& connectors, PortsOrch *portOrch, Mirr
             platform == CISCO_8000_PLATFORM_SUBSTRING ||
             platform == MLNX_PLATFORM_SUBSTRING ||
             platform == BFN_PLATFORM_SUBSTRING  ||
+            platform == CLX_PLATFORM_SUBSTRING  ||
```
Comment (prsunny) [inline orchagent/aclorch.cpp:2246]: Failing build on this defintion - CLX_PLATFORM_SUBSTRING. Please make sure associated PR is merged

Comment (Battier) [reply orchagent/aclorch.cpp:2246]: @prsunny Hi Prince, looks like the pipeline failure was fixed, could you please kindly review this PR? Thanks in advance.

**Review by prsunny — APPROVED** [review]

**Pull request merged**